### ISSUE TITLE
Hangs/Freezes after being used on Payday 2

### ISSUE THREAD

**Issue by ThePenitence** [Codeusa/Borderless-Gaming#419]
I've used BG for some time now and I've never seen it freeze/not respond, until I started playing Payday 2 this week and used it on that since the game doesn't have a borderless option. Now every time I'm done playing the game the window for BG won't respond to any clicks at all, and creates more instances of itself in the system tray, and also won't do any automatic borderless afterwards until I start a new process.

**Comment by Masterfireheart** [comment]
This is happening for me as well, both on 9.5.5 and 9.5.4, I don't play Payday 2 either, but I have to kill and restart BG randomly after opening/closing another program for it to stop hanging.

**Comment by shovas** [comment]
+1

**Comment by Chaython** [comment]
9.5.4 to 9.5.5 is only a certificate change [if he didn't update the certificate the application would be blocked by smart screen etc]
use an older build or software that is supported
https://westechsolutions.net/sites/WindowedBorderlessGaming/
That dev. replies to every question in their steam group.
This dev just trolls.

**Comment by HalfOfAKebab** [comment]
Same issue here on 9.5.5 and 9.5.6. Doesn't affect just Payday 2, but it happens randomly, even if I haven't launched any of the games that BG touches since it's been open. I can't even right-click it in the tray to close it, as the context menu doesn't respond to clicks either (the cursor just shows the hourglass icon as if it's Not Responding, but it isn't Not Responding).
